model_servers/llamacpp_python/README.md

```diff
@@ -22,39 +27,26 @@ At the time of this writing, 2 models are known to work with this service
 - **Mistral-7b**
     - Download URL: [https://huggingface.co/TheBloke/Mistral-7B-Instruct-v0.1-GGUF/resolve/main/mistral-7b-instruct-v0.1.Q4_K_M.gguf](https://huggingface.co/TheBloke/Mistral-7B-Instruct-v0.1-GGUF/resolve/main/mistral-7b-instruct-v0.1.Q4_K_M.gguf)
 
+It is suggested you place models in the [models](../../models/) directory. As for retrieving them, either use `wget` to download them with the download links above, or call the model names from the Makefile.
+
 ```bash
-cd ../models
-wget <Download URL>
-cd ../
+cd ../../models
+curl -sLO <Download URL> 
+cd model_servers/llamacpp_python
 ```
 
-or
+or:
 
 ```bash
-make -f Makefile models/mistral-7b-instruct-v0.1.Q4_K_M.gguf
+make -f Makefile mistral-7b-instruct-v0.1.Q4_K_M.gguf
```

Comment:
Make mistral and llama variables defined above.

Reply:
Does this implementation look acceptable? I feel like users should be able to trigger a download without knowing the full model name